### ISSUE TITLE
toggle comment command wrong shortcut

### ISSUE THREAD

**Issue by goshander** [kasecato/vscode-intellij-idea-keybindings#294]
current command for toggle line comment use wrong shortcut

```json
{
    "key": "ctrl+/",
    "command": "editor.action.commentLine",
    "when": "editorTextFocus && !editorReadonly"
}
```

need to be more independent of user locale and keyboard layout
```json
{
    "key": "ctrl+[Slash]",
    "command": "editor.action.commentLine",
    "when": "editorTextFocus && !editorReadonly"
}
```

**Comment by kasecato** [comment]
I released it on v1.5.10 now. Please check it

**Comment by nyurik** [comment]
I just tried it using Dvorak keyboard layout, and `ctrl+z` is broken :(  I filed #307
